util/generate-fsx-substack.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+
+import troposphere.ec2 as ec2
+from troposphere import And, Condition, Equals, If, Not, NoValue, Output, Parameter, Ref, Select, Template
+
+
+def main(args):
+    t = Template()
+
+    # [0 shared_dir, 1 efs_fs_id, 2 performance_mode, 3 efs_kms_key_id,
+    # 4 provisioned_throughput, 5 encrypted, 6 throughput_mode, 7 exists_valid_mt]
+    fsx_options = t.add_parameter(
+        Parameter(
+            "FSXOptions",
+            Type="CommaDelimitedList",
+            Description="Comma separated list of efs related options, 4 parameters in total",
```

Comment:
7 params

Reply:
fixed, also `efs -> fsx`

Reply:
minor: It seems still there, also efs. Did you miss to push the commit?

Reply:
pushed